main.py

```diff
@@ -8,7 +8,7 @@ def create_app(app_name='MYAPPNAME'):
 
 if __name__ == '__main__':
     app = create_app()
-    app.run(debug=True, host='0.0.0.0', port=int(os.environ.get('PORT', 8080))) # nosemgrep
+    app.run(debug=True, host='0.0.0.0', port=int(os.environ.get('PORT', 8080)))  # nosem
```

Comment:
<p><b style="color:black" >Rule</b>: debug enabled<br>
    <b style="color:black" >Severity</b>:&nbsp;<img src="https://cidersecurity-public-icons.s3.amazonaws.com/prSeverityIcons/high.svg" alt="HIGH" width="62px" align="center"><br>
    <b style="color:black" >Scanner:</b> <a href="https://semgrep.dev/">SEMGREP</a></p>
    <details>
    <summary><b style="color:black">More Info</b></summary>
    Detected Flask app with debug=True. Do not deploy to production with this flag enabled
as it will leak sensitive information. Instead, consider using Flask configuration
variables or setting 'debug' using system environment variables.

    </details>